### ISSUE TITLE
Unhandled Intrinsic Function in !Sub Variable Map

### ISSUE THREAD

**Issue by ronkorving** [martysweet/cfn-lint#195]
> Message: Unhandled Intrinsic Function 0, this needs implementing. Some errors might be missed.

I'm not 100% sure what this means, but it hits on this line in my CFN template file:

```yaml
      ResourceId: !Sub
        - service/${EcsClusterName}/${EcsServiceName}
        - EcsClusterName: !Ref rEcsCluster
          EcsServiceName: !GetAtt rEcsService.Name
```

Does this imply that the `Sub` function is not yet supported by cfn-lint? According to https://github.com/martysweet/cfn-lint/issues/102#issuecomment-379172620 it's supposed to be there. Thanks for a great tool, and I hope someone can give me a hint on what to do with this warning.

**Comment by martysweet** [comment]
Sorry @ronkorving for the late response! Did you manage to get this solved?

**Comment by ronkorving** [comment]
@martysweet Thanks for looking into this!
